src/main/scala/io/sdkman/vendor/release/routes/Validation.scala

```diff
@@ -45,4 +46,35 @@ trait Validation {
       17 >= version.length,
       ApiResponse(400, s"Version length exceeds 17 chars: $version").toJson.compactPrint
     )
+
+  def validateChecksumAlgorithms(checksums: Option[Map[String,String]]): Directive0 =
+    checksums
+      .map { checksum =>
+        val algorithms: Seq[String] = Seq(MD5.id, SHA1.id, SHA224.id, SHA256.id, SHA384.id, SHA512.id)
```

Comment:
I'd move this outside of the `def` declaration to save it from being evaluated each time the def is called.